lib/route.js

```diff
@@ -128,10 +128,10 @@ function buildRouting (options) {
     const opts = { ...options }
 
     throwIfAlreadyStarted('Cannot add route when fastify instance is already started!')
-
+    // Validation for array of http Methods if the route has more than one
     if (Array.isArray(opts.method)) {
       /* eslint-disable no-var */
-      for (var i = 0; i < opts.method.length; i++) {
+      for (let i = 0; i < opts.method.length; i++) {
```

Comment:
Please restore this, it was a `var` on purpose.

Reply:
Sure, Just curious  was there any particular need for this reference

Reply:
I will close the PR.

Reply:
using let in for loop is still slower than using `var`.

Reply:
oh sorry 😁 , i  forgot about the v8 perf diff with var and let